### DISCUSSION TITLE
System not recognizing a keyboard mapping

### DISCUSSION THREAD

**Discussion by GenJoeBishop** [WhiteMagic/JoystickGremlin#553] [Q&A]
Hi.  I'm trying to configure a button press on my joystick to send a keyboard command usable by Discord and Wingman.ai as a PTT command.  I'm using Subliminal's JG config file ( I can't attach an .xml file), HID Hide, and VJoy.  Everything is working well in-game, but no matter what I've done, I can't get the key mapping to appear in anything.  I've even mapped the button to a simple letter, and can't get it to register in Notepad.  What am I missing?

**Comment by WhiteMagic** [comment]
Without knowing what exactly you've done it's impossible to say what goes wrong. I would verify that on an empty profile you can get the mapt to keyboard behave as you want. Also note that you will never get a stream of characters appear in notepad or similar as it would happen when holding down a physical key, as the repeated key presses are a feature of windows and not the keyboard.
